spec/models/validations/sales/sale_information_validations_spec.rb

```diff
@@ -204,190 +204,71 @@
   end
 
   describe "#validate_discounted_ownership_value" do
-    let(:record) { FactoryBot.build(:sales_log, mortgage: 10_000, deposit: 5_000, value: 30_000, ownershipsch: 2, type: 8, saledate: now) }
```

Comment:
These changes look bigger than they are because of a formatting change.

In essence: it's not actually possible for both discount and grant to be routed to - I've removed tests that depend on this and replaced with relevant tests around the new tolerance. I've also removed the time freeze.